cartoframes/datasets.py

```diff
@@ -158,11 +158,21 @@ def _get_read_query(self, table_columns, limit=None):
 
         return query
 
-    def _get_table_columns(self):
+    def get_table_columns(self):
         """Get column names and types from a table"""
         query = 'SELECT * FROM "{schema}"."{table}" limit 0'.format(table=self.table_name, schema=self.schema)
         return get_columns(self.cc, query)
 
+    def get_table_column_names(self, exclude=None):
+        """Get column names and types from a table"""
+        query = 'SELECT * FROM "{schema}"."{table}" limit 0'.format(table=self.table_name, schema=self.schema)
+        columns = get_columns(self.cc, query).keys()
+
+        if exclude and isinstance(exclude, list):
+            columns = list(set(columns) - set(exclude))
```

Comment:
We should ensure that `exclude` is a list or tuple and not a str because:
```python
>>> exclude = 'the_geom_webmercator'
>>> set(exclude)
{'_', 'a', 'b', 'c', 'e', 'g', 'h', 'm', 'o', 'r', 't', 'w'}
```

Reply:
yep, we are already checking that it is an instance of `list`. Anyway [we'll add proper docs](https://github.com/CartoDB/cartoframes/issues/578) once all these new methods are consolidated.

Reply:
Ha, I missed the `isinstance(exclude, list)` above 🤦‍♂️ Sorry for the noise!

Reply:
np 😅